Bio/AlignIO/EmbossIO.py

```diff
@@ -204,7 +202,7 @@ def __next__(self):
                 pass
             else:
                 print(line)
-                assert False
+                raise ValueError("Unrecognised EMBOSS pairwise alignment record format.")
```

Comment:
It would be even better to remove the ``print(line)`` and put that into the message, e.g.

```
raise ValueError("Unrecognised EMBOSS pairwise line: %r\n" % line)
```